src/components/teasers/grid/teaser-grid.tsx

```diff
@@ -25,13 +39,26 @@ const StyledTeaserGrid = styled.div<{ amountOfChildren: number }>`
   gap: 24px;
 
   justify-items: stretch;
+
+  /**
+   * we give the last element 24px more space for the padding 
+   * -> distance to outer window
+   */
   grid-template-columns: ${(props) =>
-    `repeat(${props.amountOfChildren}, minmax(224px, 1fr))`};
+    `repeat(${props.amountOfChildren - 1}, minmax(224px, 1fr)) 248px`};
```

Comment:
isn't it possible to use `auto-fill` and friends here? Less explicit but we can get a similar behavior I bet.
https://css-tricks.com/auto-sizing-columns-css-grid-auto-fill-vs-auto-fit/

Reply:
Couldn't make it work with `auto-fill`

Reply:
another PR another try I guess. Let's try to merge this baby first.